test/integration/instance_with_config_file/controls/www.rb

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+http_address = attribute('http_address')
+port = attribute('http_port')
+
+# Test disabled due to flappiness - because this makes live network requests, it is prone to sporadic failure.
```

Comment:
Let's just remove this test if it's too flaky.

Reply:
This test was copied from instance_with_attached_disk, which exposes many of the same outputs specifically to enable this test. I haven't actually run this test at all. I assumed that either we should have it in both places or remove it in both places, but I wasn't sure if there was an issue in the backlog to clean up the technical debt.

I could probably add a retry loop to make it less flaky, but I don't know how to quickly judge the flakiness before PR merge.

Do you want me to remove it here AND remove the outputs it needed, or also remove it from the other example test(s), or try to make it more robust (and take longer to fail)?

Reply:
Incremental improvement, try to leave the codebase better than you found it. You can leave the old one but no need to copy over the commented code here.